cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,25 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io');
 
+    cy.get(':nth-child(2) > .nav-link').click();
+
+    cy.get('.navbar-brand').should('exist');
+    cy.get(':nth-child(1) > .nav-link').should('exist');
+    cy.get(':nth-child(2) > .nav-link').should('exist');
+    cy.get(':nth-child(3) > .nav-link').should('exist');
+    cy.get('h1.text-xs-center').should('exist');
+    cy.get('p.text-xs-center > a').should('exist');
+    cy.get(':nth-child(1) > .form-control').should('exist');
+    cy.get(':nth-child(2) > .form-control').should('exist');
+    cy.get('.btn').should('exist');
+
+    cy.get('input[type="email"]').type('xokat@mailinator.com');
```

Comment:
Avoid hardcoding credentials in the test. Use variables or a config file to store and retrieve sensitive information.